x-pack/plugins/endpoint/public/applications/endpoint/types.ts

```diff
@@ -75,17 +75,67 @@ export interface PolicyListState {
 }
 
 /**
- * Policy list store state
+ * Policy details store state
  */
 export interface PolicyDetailsState {
   /** A single policy item  */
-  policyItem: PolicyData | undefined;
+  policyItem?: PolicyData;
   /** data is being retrieved from server */
+  policyConfig?: PolicyDetailsConfig;
   isLoading: boolean;
   /** current location of the application */
   location?: Immutable<EndpointAppLocation>;
 }
 
+export interface PolicyDetailsConfig {
+  windows: WindowsPolicyConfig;
+  mac: MacPolicyConfig;
+  linux: LinuxPolicyConfig;
+}
+
+export interface WindowsPolicyConfig {
+  malware: {
+    mode: string;
+  };
+  eventing: {
+    process: boolean;
+    network: boolean;
+  };
+}
+
+export interface MacPolicyConfig {
```

Comment:
Can you please add doc comments?
* Ideally add a doc comment to each field in a type/interface
* Ideally add a doc comment to anything exported.